src/main.ts

```diff
@@ -205,6 +206,11 @@ const command: Command = {
 			describe: 'Disable lazy widget loading detection',
 			type: 'boolean'
 		});
+
+		options('serviceWorker', {
```

Comment:
What does that consist of specifically? The tests for dojorc that I've seen all involve loading arguments defined here from a configuration file.